webroot/src/components/App/App.ts

```diff
@@ -73,6 +78,14 @@ class App extends Vue {
     //
     // Methods
     //
+    async setCurrentCompact() {
+        const compact = CompactSerializer.fromServer({ type: CompactType.ASLP }); // Temp until server endpoints define this
```

Comment:
The user's compact. I'll update the comment.

Reply:
@jsandoval81 Was this updated? Unsure if this is resolved or not.

Reply:
Yeah, GitHub UI is a little quirky with that. It preserves the code here ☝️ as it was when you commented; and then puts a little "Outdated" tag on top of the comment if the code gets updated. AFIAK you have to go to the "Files Changed" tab to see the latest. I kinda get that they are trying to preserve the state of things so that the original comment stays in context. I just wish they had a link or a preview for the updated code.